examples/wildfly-keycloak-saml-adapter/src/test/java/org/jboss/intersmash/examples/wildfly/keycloak/saml/KeycloakCapableImageBasedWildflyApplicationTest.java

```diff
@@ -0,0 +1,54 @@
+package org.jboss.intersmash.examples.wildfly.keycloak.saml;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+
+import org.apache.http.HttpStatus;
+import org.jboss.intersmash.annotations.Intersmash;
+import org.jboss.intersmash.annotations.Service;
+import org.jboss.intersmash.annotations.ServiceUrl;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+@Intersmash(@Service(KeycloakCapableImageBasedWildflyApplication.class))
+public class KeycloakCapableImageBasedWildflyApplicationTest {
```

Comment:
@fabiobrz I don't understand how is this supposed to work when we don't have an actual Keycloak server....

Reply:
As discussed earlier, this is a WildFly service example, no Keycloak in place.
The only thing related to Keycloak is that the configuration of the Galleon-provisioned server includes the Keycloak SAML Adapter Galleon layer.